platforms/corda-ent/charts/generate-pki/templates/configmap.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: {{ .Values.nodeName }}-pkitool-conf
   namespace: {{ .Values.metadata.namespace }}
   labels:
-    app.kubernetes.io/name: {{ .Values.nodeName }}-cenm-conf
+    app.kubernetes.io/name: {{ .Values.nodeName }}-pkitool-conf
```

Comment:
Please change the corresponding check for generate-pki too in roles/setup/signer